lib/api/_loaders/_base-loader.js

```diff
@@ -145,6 +145,15 @@ class BaseLoader extends EventEmitter {
     return this.nightwatchInstance.settings.disable_typescript;
   }
 
+  get isTraceEnabled() {
+    return this.nightwatchInstance.settings.trace.enabled;
+  }
```

Comment:
we can use `this.settings.trace.enabled`

Reply:
this is not needed anymore.